ProcessMaker/Http/Controllers/Process/ModelerController.php

```diff
@@ -135,7 +135,7 @@ public function prepareModelerData(ModelerManager $manager, Process $process, Re
             'countScriptCategories' => $countScriptCategories,
             'isProjectsInstalled' => $isProjectsInstalled,
             'isPackageAiInstalled' => $isPackageAiInstalled,
-            'isAiGenerated' => request()->query('ai'),
+            'isAiGenerated' => request()->query('isAiGenerated'),
```

Comment:
Please keep the variable `ai` instead of `isAiGenerated`, this change could be affect our LucidChart integration